### PR TITLE
Log when a message is updated (edited) or deleted.

### PR DESCRIPTION
**Pull request by LamboCreeper**
Added logging for when a user updates or deletes a message. I feel like there could be a better way to write the actual event handler, rather than having to do something like:
```js
client.on("messageDelete", (message) => {
    events.messageDelete.run(message);
});
```
every time you want to add a new event... Suggestions welcome.

### CODE REVIEW

File: events/messageDelete.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+// The code that runs when the event is executed.
+function run(message) {
+	const logsChannel = message.guild.channels.find((channel) => {
+		return channel.id == config.logs_channel;
+	});
+	const embed = new Discord.RichEmbed();
+
+	embed.setTitle("Message Deleted");
+	embed.setDescription(`Author: ${message.author}\nChannel: ${message.channel}`);
```
Comment (jacobgasser) [inline events/messageDelete.js:15]: This won't show who actually deleted the message, just the author.

**Review by jacobgasser — CHANGES_REQUESTED** [review]

**Comment by jacobgasser** [comment]
There are other ways to setup the event handler but I think this way is the best because of how explicit it is. Other ways would make it not so obvious.

File: events/messageDelete.js (hunk 1 above)
Comment (saramaebee) [reply events/messageDelete.js:15]: It is possible to access the executor by way of `GuildAuditLogsEntry.executor`, feel free to make the change

Comment (LamboCreeper) [reply events/messageDelete.js:15]: I can't seem to figure this out. If one of you would like to combat it that would be great. We don't have this feature currently, so I'm not too bothered about it not being in MVP.

File: events/messageUpdate.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	if (![oldMessage.content, newMessage.content].includes("")) {
+		const logsChannel = oldMessage.guild.channels.find((channel) => {
+			return channel.id == config.logs_channel;
+		});
+		const embed = new Discord.RichEmbed();
+
+		embed.setTitle("Message Updated");
+		embed.setDescription(`Author: ${oldMessage.author}\nChannel: ${oldMessage.channel}`);
+		embed.addField("Old Message", oldMessage.content || ".");
+		embed.addField("New Message", newMessage.content || ".");
```
Comment (jacobgasser) [inline events/messageUpdate.js:18]: I don't like how it is a period. Replace it with something more obvious as to what it is.

**Review by jacobgasser — CHANGES_REQUESTED** [review]

**Review by jacobgasser — APPROVED** [review]

**Pull request merged**